### DISCUSSION TITLE
"AHR-ti-zin" or "Ahr-TEE-zhun"?

### DISCUSSION THREAD

**Discussion by fatrabbit-la** [artisan-roaster-scope/artisan#1152] [General]
Ok, I know this seems silly but Kyle Rowsell recently posted a video in which he referred to our favorite roasting software as, "Ahr-TEE-zhun". I called him out on it and he basically said, "Yup, that's what I said."

I've always said and heard it pronounced as "AHR-ti-zin" but I don't want to assume.

Can the higher ups please confirm definitively the correct pronunciation?

**Comment by poundy** [comment]
"san" as in SAND.

I'm an Aussie, so our pronunciation of most things is correct * 

* if you ask us.

OK so I don't know there is such a thing as "correct" here. Local language and accents will muddy whatever water you happen to be bathing in at the time.

**Comment by fatrabbit-la** [comment]
Clearly I have nothing better to do but I think I’m getting to the bottom of why Kyle’s pronunciation bugs me. Yes, you’re right, local language and accents can put a different verbal spin on things…

https://www.youtube.com/watch?v=6p1TcdnUEX4 <https://www.youtube.com/watch?v=6p1TcdnUEX4>

…but he pronounces it as if to say artesian, which has to do with the flow of water from underground to the surface or referencing a type of well whereas artisan, and the unmistakable pronunciation variations (see link above) refers in a broad sense to someone who practices a trade or handicraft.

I don’t pour water into my drum nor do I siphon it into the cooling tray post roast from underneath my garage floor so I can only assume the intent is artisan and not even close to artesian.

Just a bit of fun but I’m trying to build up some ammo before I further tease Kyle on his YouTube page. And then I’ll stop wasting everyone’s time.

Dan

Fat Rabbit Coffee
***@***.***
www.fatrabbitcoffee.com

**Reply by poundy** [reply]
artesian is defo wrong :)
